src/Uno.UI/UI/Xaml/DependencyPropertyDetails.cs

```diff
@@ -29,8 +29,9 @@ private DependencyPropertyDetails()
 			_stack = _pool.Rent(MaxIndex + 1);
 		}
 
-		~DependencyPropertyDetails()
+		internal void Dispose()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either implement 'IDisposable.Dispose', or totally rename this method to prevent confusion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5247895)